src/governance.rst

```diff
@@ -149,10 +153,10 @@ the time when a vote is called, but may never be less than 24 hrs.
     for voting on the GH issue in question.
     The voting must period be open for at least one core
     member meeting cycle to allow for clarification questions
-    and discussions.
+    and discussions. Friendly reminders to vote are encouraged.
 
     * Standard
-    * 50% Majority to pass
+    * 50% plus one of those voting to pass
```

Comment:
Stupid clarity point: suppose 5 people vote. 50% is 2.5; 50% plus one is 3.5. I don't think that we want a 3/5 vote to fail. Maybe say instead `Strict majority` or `Over 50%` or something like that?

Reply:
Yeah, that is better.